android/expoview/build.gradle

```diff
@@ -337,5 +337,10 @@ dependencies {
 
   // expo-av
   // See explanation in expo-av/build.gradle
-  api 'expolib_v1.com.google.android.exoplayer:expolib_v1-extension-okhttp:2.6.1'
+  api 'com.google.android.exoplayer:extension-okhttp:2.9.2'
+
+  // This can be removed when sdk 32 is dropped
+  api 'expolib_v1.com.squareup.okhttp3:okhttp:3.6.0'
```

Comment:
These can be `provided` as we don't want to include them in ejected apps (we want to include them only in Expo Client and it's the old ABIs which will depend on them strongly).